README.md

```diff
@@ -2,4 +2,159 @@
+
+### Twenty-One
+
+[Blackjack](https://en.wikipedia.org/wiki/Blackjack), also known as "twenty-one", is a casino card game in which a player attempts to build a hand that is closer to a score of 21 than the house (or dealer's) hand without going over a total of 21 (or "busting"). The Ace's base value is one (1), but can be used as a value of eleven (11) if doing so does not bust a hand. Players are initially dealt two cards and choose to be dealt additional cards (to "hit") or to hold their hand until the end of the round (to "stay").
+
+## Instructions
+
+Open `swift-blackjack.xcworkspace`. 
+
+You may deselect the target membership of the testing files in order to work on the classes-one-by-one. This option is found in the File Inspector pane of the Utilities area when the relevant file is selected in the Project Navigator pane of the Navigator area.  
+**Note:** *Doing this may report false-positives to your Learn.co profile's local build light, so make sure you have test results for all five custom classes before regarding your solution as complete.*
+
+### `Card.swift`
+
+1. Re-enable the target membership for `CardSpec.swift` for `swift-blackjackTests`.
+
+2. Create a class file for the `Card` class. It will need the same properties as the `FISCard` class from `objc-blackjack`:
+  * `suit`, a string,
+  * `rank`, a string,
+  * `cardLabel`, a string, and
+  * `cardValue`, an unsigned integer.
+
+3. Write an initializer for `Card` that takes two arguments for `suit` and `rank`.
+  * It should also set the `cardLabel` property to string containing both the suit and the rank in the form of `♠︎A` (for the Ace of Spades).
+  * It should also set the `cardValue` property appropriately for scoring Blackjack: Aces are worth one, number cards are worth their face values, and face cards are worth ten.
+
+4. Make the `Card` class conform to the `CustomDebugStringConvertible` protocol. A `Card` object should print its `cardLabel` string to the console when it is submitted to the `print()` function.
+
+5. Set up two class methods called `validSuits` and `validRanks` that return arrays containing the four suit icons and the thirteen string representations of the ranks (Ace through King).
+
+### `Deck.swift`
+
+1. Re-enable the target membership for `DeckSpec.swift` for `swift-blackjackTests`.
+
+2. Create a class file for the `Deck` class. It should have two private array properties for holding cards: one for holding the remaining (undealt) cards, and the other for holding the dealt cards. The test file cannot see them so name them yourself.
+
+3. Write an initializer that generates the fifty-two unique cards to represent a standard deck of playing cards and holds them in the array of cards that can be dealt.
```

Comment:
it's that numbers less than ten should be spelled out: http://wjea.org/wp-content/uploads/2014/02/WJEA-AP-Style-Cheat-Sheet.pdf

it annoys me that "Standard 52-card deck" is how it's written on Wikipedia. Being a compound noun, it's my intuition to spell it out. <sigh>  https://en.wikipedia.org/wiki/Standard_52-card_deck